frontend/taipy-gui/src/components/Taipy/Menu.tsx

```diff
@@ -86,9 +85,10 @@ const Menu = (props: MenuProps) => {
 
     useEffect(() => {
         if (props.selected) {
-            setSelectedValue(props.selected);
+            const updatedSelected = [...props.selected, currentSelection];
```

Comment:
what if currentSelection is in props.selected or currentSelection is not defined ?

Reply:
this part has been removed in the latest commit, and being handle in the useCallback